Broken initial state

shashkovdanil opened the pull request.

shashkovdanil committed:
Fix broken component and add correctly render test

shashkovdanil committed:
rename variable

skoob13 commented:
Hey, thank you for a PR! Can you provide any additional information? Looks like 1.2.2 ver. works for me.

shashkovdanil commented:
I added a test that checks the rendering. Also updated the dependencies and added the initialState. Just in case, added a check for the existence of the instance where it is used

a-tokyo committed:
✏️Fix typo `Booleanean` -> `Boolean`

shashkovdanil committed:
Merge pull request #1 from A-Tokyo/patch-1

✏️Fix typo `Booleanean` -> `Boolean`
